pkg/sriov/sriov.go

```diff
@@ -272,6 +246,12 @@ func (s *sriovManager) ResetVFConfig(conf *sriovtypes.NetConf) error {
 
 	// Set vlan to 0
 	if conf.Vlan != 0 {
+		if conf.VlanQoS != 0 {
+			if err = s.nLink.LinkSetVfVlanQos(pfLink, conf.VFID, 0, 0); err != nil {
+				return fmt.Errorf("failed to set vf %d vlan: %v", conf.VFID, err)
+			}
+		}
+
 		if err = s.nLink.LinkSetVfVlan(pfLink, conf.VFID, 0); err != nil {
```

Comment:
Should this be under an `else` block to avoid resetting vlan twice?

Reply:
Yes, it should. I'll fix it.